### PR TITLE
feat: create guidelines for an issue and getting help

### PR DESCRIPTION
**Pull request by Midnighter**
* Add a GitHub issue template and a convenience function to display the status and
version of important dependencies. (You can see a very similar setup when you go to https://github.com/pandas-dev/pandas/issues/new.)
* Add a link to a guide on how to ask a good question.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+```python
+# Paste your code here.
+
+```
+
+#### Actual Output
+
+#### Expected Output
+
+#### Problem description
```
Comment (hredestig) [inline .github/ISSUE_TEMPLATE.md]: I'd like to start with problem description, suggest to move it to the top

File: README.rst [hunk 2]
```diff
@@ -36,10 +36,14 @@ also be
 `downloaded <https://readthedocs.org/projects/cobrapy/downloads/>`_.
 
 Please use the `Google
-Group <http://groups.google.com/group/cobra-pie>`_ for help.
+Group <http://groups.google.com/group/cobra-pie>`_ for help. When posting
+questions, please refer to these `StackOverflow
+guidelines <https://stackoverflow.com/help/how-to-ask>`_. You will make our
```
Comment (hredestig) [inline README.rst]: 'By writing a well formulated question, with sufficient detail, you are much more likely to quickly receive a good answer!'

File: cobra/util/version_info.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+from __future__ import absolute_import, print_function
+
+from builtins import dict
+
+import platform
+
+import pip
+
+__all__ = ("print_info",)
```
Comment (hredestig) [inline cobra/util/version_info.py]: show_versions ?

File: cobra/util/version_info.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```
Comment (hredestig) [inline cobra/util/version_info.py]: perhaps should add a couple of trivial tests for this as well?

File: cobra/util/version_info.py (hunk 3 above)
Comment (Midnighter) [inline cobra/util/version_info.py]: Yep, should have used the refactor functionality :P

File: cobra/test/test_util.py [hunk 5]
```diff
@@ -293,3 +297,58 @@ def test_union(self, dict_list):
         # should only add 1 element
         assert len(test_list) == 2
         assert test_list.index("test2") == 1
+
+
+class TestVersionInfo:
```
Comment (hredestig) [inline cobra/test/test_util.py]: wow :)

**Pull request merged**